src/SamplesApp/UITests.Shared/Windows_ApplicationModel/Calls/PhoneCallHistoryEntryReader.xaml.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Uno.UI.Samples.Controls;
+using Windows.Data.Xml.Dom;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Notifications;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+using Windows.ApplicationModel.Calls;
+
+using Windows.ApplicationModel.Calls;
+
+namespace UITests.Shared.Windows_ApplicationModel.Calls
+{
+	[SampleControlInfo("Windows.ApplicationModel.Calls", "PhoneCallHistoryReader")]
+
+	public sealed partial class PhoneCallHistoryEntryReader : UserControl
+	{
+
+		public PhoneCallHistoryEntryReader()
+		{
+			this.InitializeComponent();
+		}
+
+		private async void getCallLog_Click(object sender, RoutedEventArgs e)
+		{
+			uiErrorMsg.Text = "";
+			uiLastNumber.Text = "";
+			uiLastDate.Text = "";
+			uiOkMsg.Text = "";
+			uiDuration.Text = "";
+
+			PhoneCallHistoryStore oCallHist;
+			try
+			{
+				oCallHist = await PhoneCallHistoryManager.RequestStoreAsync(PhoneCallHistoryStoreAccessType.AllEntriesLimitedReadWrite);
+			}
+			catch (Exception ex)
+			{
+				uiErrorMsg.Text = "Exception while RequestStoreAsync: " + ex.Message;
```

Comment:
I'm still getting a null reference exception here. Let's show the full trace:
```suggestion
				uiErrorMsg.Text = "Exception while RequestStoreAsync: " + ex;
```